homeassistant/components/seventeentrack/icons.json

```diff
@@ -2,32 +2,33 @@
   "entity": {
     "sensor": {
       "not_found": {
-        "default": "mdi:package"
+        "default": "mdi:package-variant-closed"
       },
       "in_transit": {
-        "default": "mdi:package"
+        "default": "mdi:package-variant-closed"
       },
       "expired": {
-        "default": "mdi:package"
+        "default": "mdi:package-variant-closed-remove"
       },
       "ready_to_be_picked_up": {
-        "default": "mdi:package"
+        "default": "mdi:package-variant-closed"
       },
       "undelivered": {
-        "default": "mdi:package"
+        "default": "mdi:package-variant-closed"
       },
       "delivered": {
-        "default": "mdi:package"
+        "default": "mdi:package-variant-closed-check"
       },
       "returned": {
-        "default": "mdi:package"
+        "default": "mdi:package-variant-closed-remove"
       },
       "package": {
-        "default": "mdi:package"
+        "default": "mdi:package-variant-closed"
       }
     }
   },
```

Comment:
These changes are not related to this PR and should be removed from it for that reason.

Reply:
I'll do a separate PR for those. As there is no `mdi:package-plus`, I've just used `mdi:package` for now.